test/unit/utility/views/CMakeLists.txt

```
@@ -1,3 +1,5 @@
 add_subdirectories()
 
+seqan3_test(repeat_n_test.cpp)
+seqan3_test(repeat_test.cpp)
 seqan3_test (zip_test.cpp)
```

As you touch this file, you could remove this space
```suggestion
seqan3_test(zip_test.cpp)
```

---

In cmake we do space before `(` xD